Common/Data/SubscriptionManager.cs

```diff
@@ -128,13 +136,13 @@ public SubscriptionDataConfig Add(Type dataType, TickType tickType, Symbol symbo
             var newConfig = new SubscriptionDataConfig(dataType, symbol, resolution, dataTimeZone, exchangeTimeZone, fillDataForward, extendedMarketHours, isInternalFeed, isCustomData, isFilteredSubscription: isFilteredSubscription, tickType: tickType);
 
             //Add to subscription list: make sure we don't have this symbol:
-            if (_subscriptions.ContainsKey(newConfig))
+            if (_dataManager.SubscriptionManagerContainsKey(newConfig))
```

Comment:
NOTE: These method namings are purposeful so that as we continue this refactoring/redesign we're very aware of what/where things are/coming from. When finalized, we'll go back through and take naming/API considerations into account, until then, it's scaffolding